shared/media-accelerator/develop/_media-stream-encryption.mdx

```diff
@@ -0,0 +1,93 @@
+import * as data from '@site/data/variables';
+import Prerequites from '@docs/shared/common/prerequities.mdx';
+import ProjectSetup from '@docs/shared/media-accelerator/develop/encrypt-media-streams/project-setup/index.mdx';
+import ProjectImplement from '@docs/shared/media-accelerator/develop/encrypt-media-streams/project-implementation/index.mdx';
+import ProjectTest from '@docs/shared/media-accelerator/develop/encrypt-media-streams/project-test/index.mdx';
+import Reference from '@docs/shared/media-accelerator/develop/encrypt-media-streams/reference/index.mdx';
+
+
+Media stream encryption ensures that only authorized users in a channel see and hear each other. This prevents potential eavesdroppers from accessing sensitive and private information shared in a channel. <Vpd k="NAME"/> provides built-in encryption methods that you can use to guarantee data confidentiality during transmission, when required.
+
+This page shows you how to integrate built-in media stream encryption into your <Vpl k="CLIENT" /> using <Vpd k="NAME"/>.
+
+## Understand the tech
+
+To ensure secure communication, your <Vpl k="CLIENT" /> uses an SSL [key](https://en.wikipedia.org/wiki/Key_authentication) and [salt](https://en.wikipedia.org/wiki/Salt_(cryptography)) to encrypt and decrypt data in the channel. You use the key and salt to create an encryption configuration. <Vg k="AGORA_BACKEND" /> uses the encryption configuration to encrypt a stream and sends it to remote users. When a remote user receives an encrypted media stream, the remote <Vpl k="CLIENT" /> decrypts it using the same salt and key.
+
+The following figure shows the call flow for media stream encryption:
+
+![Encrypt media stream](/images/common/media-stream-encryption.png)
+
+All users in a channel must use the same encryption configuration to initiate `agoraRtcService` and enable encryption before joining a channel. If you don’t have the correct configuration, you cannot decrypt channel content. Best practice is to have your authentication system generate a new key and salt regularly.
```

Comment:
must use the same encryption configuration. You set this up when you initiate the Agora Engine and enable ...

Reply:
Fixes